src/App-custom-function.spec.ts

```diff
@@ -0,0 +1,277 @@
+import 'mocha';
+import { assert } from 'chai';
+import rewiremock from 'rewiremock';
+import sinon, { SinonSpy } from 'sinon';
+import { FunctionsCompleteErrorResponse, FunctionsCompleteSuccessResponse, WebClientOptions } from '@slack/web-api';
+import App from './App';
+import { Override, mergeOverrides } from './test-helpers';
+import { FunctionInputs, Receiver, ReceiverEvent } from './types';
+
+class FakeReceiver implements Receiver {
```

Comment:
I'm a big supporter of breaking the receiver and mock spies into testing utilities of some sort. Solid suggestions. I have a few other refactors in mind and might want to save these for another PR to avoid too many changes (I'd want to update all `*.spec.ts` files for this)!